packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -427,18 +428,28 @@ class _ProductQueryPageState extends State<ProductQueryPage>
   }
 
   Future<String?> _getTranslatedCountry() async {
-    if (_country == null) {
+    if (_country == null) return null;
```

Comment:
```suggestion
    if (_country == null) {
      return null;
    }
```
Please don't change the format!